modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/310_reroute_processor.yml

```diff
@@ -139,3 +168,73 @@ teardown:
           query:
             match: {"_id": "example-log"}
   - match: { hits.hits.0._source.message: "this is an error log" }
```

Comment:
Can you add to the pipelines and to this test clause so that there's the equivalent of these lines associated with this test, too?

```
  - match: { _source.added-in-pipeline-before-reroute : true }
  - match: { _source.added-in-pipeline-after-reroute : true }
```

That is, I want a `set` processor in each of the `logs-router` and `logs-nginx` pipelines, and a `match` against `hits.hits.0` for each of the values that those `set` processors set. There's no need for the `existing-field`, that's already covered here by `"this is an error log"`.